definitions/npm/react-jss_v8.x.x/flow_v0.65.x-/react-jss_v8.x.x.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+declare module 'react-jss' {
+  import type { Component, ComponentType, ElementConfig } from 'react';
```

Comment:
I'm not the original author but I would like this PR to be accepted since it would solve a number of issues for us. I Googled for flow contributing guidelines but I could only find https://github.com/flow-typed/flow-typed/wiki/Contributing-Library-Definitions which doesn't explain why that wouldn't work.

The FAQ does have a question that seems to say that this is the correct way to do the import: https://github.com/flow-typed/flow-typed/wiki/FAQs#how-do-i-build-type-definitions-that-depend-on-other-definitions . I can't find anything in the flow docs either: https://flow.org/en/docs/libdefs/creation/

Do you perhaps have a link for me?

Reply:
That's sounds about right. :)

Reply:
You're welcome to take these changes and create a new PR and run with it.

Reply:
Great, I have created a type definition based on @nathanieldamned and @clementprevot work. It's in internal code review and lacking tests at the moment but if it works for us I'll definitely make a PR!

Reply:
I guess we need a bit of synchronization on those React core types import :)  
See the discussion in #2645 where @villesau where saying the opposite.